spec/features/entity_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
- [ ] I ran your tests and got some of them failing as seen below. Please make sue all your tests are passing before you submit for another review.
![image](https://github.com/BANCUNGUYE66/budget-app/assets/45460192/38528720-2118-449c-90a2-98dd90e7dc8e)

- [ ] Please add unit tests as well of `type: :model`. I see you only created integration tests.